f5-bigip-mixin/dashboards/f5-cluster-overview.libsonnet

```diff
@@ -0,0 +1,633 @@
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+local dashboardUid = 'f5-cluster-overview';
+
+local promDatasourceName = 'datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local nodeAvailabilityPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      '100 * sum by(partition, instance) (bigip_node_status_availability_state{job=~"$job", instance=~"$instance"})  / clamp_min(count by(partition, instance) (bigip_node_status_availability_state{job=~"$job", instance=~"$instance"}),1)',
```

Comment:
general question, would it be better also include "job" across all aggregations like sum (for all dashboards)?

Reply:
yeah